CRM/CiviAwards/Upgrader/Steps/Step1011.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+use CRM_CiviAwards_Service_ApplicantManagementMenu as ApplicantManagementMenu;
+
+/**
+ * Update menus with new URL.
+ */
+class CRM_CiviAwards_Upgrader_Steps_Step1011 {
+
+  /**
+   * Runs the upgrader changes.
+   *
+   * @return bool
+   *   Return value in boolean.
+   */
+  public function apply() {
+    (new ApplicantManagementMenu())->updateCaseCategorySubmenus(
+      CRM_CiviAwards_Helper_CaseTypeCategory::AWARDS_CASE_TYPE_CATEGORY_NAME
```

Comment:
This method needs to be updated to resetCaseCategoryUrl`

Reply:
Done: https://github.com/compucorp/uk.co.compucorp.civiawards/pull/219/commits/4c7de8e1506a32ef86cb98b5c39c7a73ecc6a420